udp-rsc-offload.md

```diff
@@ -19,15 +19,14 @@ The Winsock API (currently only software fallback) already exists, and details o
 
 # TCPIP
 
-This section describes necessary updates in the Windows network stack to support URO.
-
-- If hardware URO is enabled, and a socket opts-in to URO with a max coalesced size the same as, or larger than, the hardare offload, then TCPIP will deliver the NBLs from hardware unmodified.
-- If hardware URO is enabled, but a socket opts-in to a smaller max coalesced size, TCPIP/AFD will break the coalesced receive into the smaller size for the socket.
-- If hardware URO is enabled, but a socket does not opt-in to URO, then TCPIP will resegment receives for that socket.
-- If hardware URO is not available, but a socket opts-in to URO with the Winsock API, software URO will be used for that socket. Software coalescing will follow the same rules as NDIS, except for the following:
-    - Software coalescing may decide to use multiple NBLs in a chain of up to 255, instead of a single NBL.
-    - Software coalescing will not coalesce IP fragments, broadcast or multicast datagrams, or datagrams with IP extension headers.
-- TCPIP will disable the hardware offload, and the software fallback, if any WFP/LWF filters that are URO-incompatible are installed on the system.
+This section describes necessary updates in the Windows network stack (TCPIP & AFD) when hardware URO support is enabled:
```

Comment:
Should we add a note for forwarding packets (re-segment URO if necessary, or convert to USO)? What about stuff like IPSNPI?

Reply:
Yeah, probably. What do you suggest. Do you just want to merge this and make a new PR to add stuff for that?

Reply:
I'm fine with merging this as-is, as the list of changes to TCPIP is likely to grow substantially as we discover more issues.

The other question is: how much of this do we need to publicly document, and for the truly implementation details side of things, should we track internally?